packages/forms/docs/10-upgrade-guide.md

```diff
@@ -101,3 +105,44 @@ An easy way to upgrade your code quickly is to find and replace:
 
 - `Closure $get` to `\Filament\Forms\Get $get`
 - `Closure $set` to `\Filament\Forms\Set $set`
+
+#### TextInput numeric masks are replaced with Alpine.js masks
+
+Filament v2 had a fluent closure mask syntax for managing more complex input masks. In v3 you can use Alpine.js masking instead. So instead of 
+```
+->mask(fn (TextInput\Mask $mask) => $mask
+    ->numeric() // And any other transformations needed
+)
+```
+You can do:
+```
+->mask(RawJs::make(<<<'JS'
+    $input.startsWith('34') || $input.startsWith('37') ? '9999 999999 99999' : '9999 9999 9999 9999'
+JS))
+```
+If you were using input masks for money related fields, Alpine already has [builtin helpers](https://alpinejs.dev/plugins/mask#money-inputs) for it:
+```
+->mask(RawJs::make(<<<'JS'
+    $money($input, '.', ',', 2)
+JS))
+```
+
+#### Action execution with forms
```

Comment:
Doesn't really belong in the forms upgrade guide, should probably be in a new actions upgrade guide file that you'll need to create.